bottle.py

```diff
@@ -65,7 +65,7 @@ def _cli_patch(cli_args):  # pragma: no coverage
     _cli_patch(sys.argv)
 
 ###############################################################################
-# Imports and Helpers used everywhere else #####################################
+# Imports and Helpers used everywhere else ####################################
```

Comment:
This isn't really a spelling change. I noticed it when I looked at the file (I'm considering using this project for a task...)